core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -322,28 +323,33 @@ def detect_authentication_to_install
     def detect_payment_method_to_install
       return 'paypal' if Bundler.locked_gems.dependencies['solidus_paypal_commerce_platform']
       return 'bolt' if Bundler.locked_gems.dependencies['solidus_bolt']
+      return 'braintree' if Bundler.locked_gems.dependencies['solidus_braintree']
 
-      descriptions = {
-        paypal: "- [#{set_color 'paypal', :bold}] Install `solidus_paypal_commerce_platform` (#{set_color :default, :bold}).",
-        bolt: "- [#{set_color 'bolt', :bold}] Install `solidus_bolt`.",
-        none: "- [#{set_color 'none', :bold}] Skip installing a payment method.",
+      allowable_payment_methods_hash = {
```

Comment:
Thoughts about making `allowable_payment_methods_hash` the new `PAYMENT_METHODS` constant? For the `payment_method` `class_option`, we could use `enum: Set[*PAYMENT_METHODS.values.flatten]` to render all of them. That way we wouldn't repeat information in two different places.

Reply:
I agree it would be nice to avoid repeating the payment method information. Let me see what I can do :+1: